core/src/entities/BiblePhrase.entity.ts

```diff
@@ -9,16 +9,16 @@ import {
     BeforeUpdate,
     Index
 } from 'typeorm';
-import { BibleCrossReferenceEntity, BibleNoteEntity } from '.';
+import { BibleCrossReferenceEntity } from './BibleCrossReference.entity';
+import { BibleNoteEntity } from './BibleNote.entity';
 import { generatePhraseId, parsePhraseId } from '../functions/reference.functions';
 import { PhraseModifiers, IBiblePhraseRef } from '../models';
 import { IBiblePhraseWithNumbers } from '../models/BiblePhrase';
 import { IContentPhrase } from '../models/ContentPhrase';
-import { isNode } from '../functions/utils.functions';
 
 @Entity('bible_phrase')
 export class BiblePhraseEntity implements IBiblePhraseWithNumbers {
-    @PrimaryColumn({ type: isNode && process.argv.indexOf('sqlite') === -1 ? 'bigint' : 'integer' })
+    @PrimaryColumn()
```

Comment:
this is intentional.

I defined this column as "bigint" for postgres migration and "integer" for sqlite migration

Then, the model file will adapt to either data type, since its not specified

@chriswep , if you still use MySQL, I'm happy to add a testing setup for that as well.

Reply:
for sure, thanks!

Reply:
I added mysql migrations with a reproducible docker setup, and tests